docs/faq/proof-of-stake/pos-general-faq.md

```diff
@@ -0,0 +1,123 @@
+
+Remember, your wallet has to be online and unlocked for your ticket to cast a valid vote on a block when it is selected. If you don't want to leave your wallet unlocked and online, have a look at a PoS stake pool.
+
+---
+
+#### **3. What is ticket price?**
+
+The price for tickets is determined by an algorithm. This algorithm aims to keep the ticket pool size, which is the total amount of tickets in the PoS system ticket pool, at around the target. The target is set to 40960 tickets. The ticket price will go up or down according to the demand for tickets.
+
+Every 144 blocks the algorithm adjusts the ticket price. This is called a buying window. Each block can contain 20 newly bought tickets. This means that in every buying window a maximum of 2880 tickets can be added to the PoS system ticket pool.
+
+---
+
+#### **4. What are fees?**
+
+The PoS system utilizes two types of fees, a txfee and a ticketfee.
+	
+The txfee is a fee you pay the network for handling the transaction to buy your ticket. This fee is at default set to a low amount (0.01 DCR/kB) and should not be changed.
+The ticketfee is the fee you pay if your ticket gets mined. This fee is an incentive for PoW miners to mine your ticket and add it to the ticket pool.
+
+In case the buying window is full, meaning there are more tickets being bought than there are available to be mined during that window, the ticketfee will be used to determine which tickets get put into the ticket pool. The tickets with the highest ticketfee will be selected by the PoW miners.
+
+Fees are calculated in DCR per kilobyte. Since a ticket is only about 300bytes in size, the actual fees you pay will be lower than what you set as the specific fee amount. Be aware, stake pool tickets are larger than solo-stake tickets.
+
+Both txfees and ticketfees will not be returned when a ticket votes, expires, or misses a vote.
+
+Returning the funds back to your wallet after a ticket votes or is revoked does not generate a fee.
+
+---
+
+#### **5. What is a stake pool?**
+
+A stake pool is the equivalent of a mining pool, but for the PoS system. Through the options in your DCR wallet you can give your voting rights to a stake pool. If your ticket is selected to validate a block the stake pool will cast the vote for you and you are rewarded with the PoS reward. Unlike the mining pools, the PoS reward is not split amongst the users of the stake pool, but the full reward goes to the owner of the ticket.
+
+The use of a stake pool will allow you to buy tickets with your DCR but not have your own wallet unlocked and always online.
```

Comment:
> Stake pools allow users to purchase tickets with voting rights delegated to a 3rd party to vote on their behalf.  This allows users who would otherwise be unable to run a 24/7 unlocked voting wallet to still participate in the PoS system.